### ISSUE TITLE
Change model

### ISSUE THREAD

**Issue by lunaticsm** [mrgick/duck_chat#11]
how i change the model ai from default to Gpt4.0 mini ?

**Comment by mrgick** [comment]
Hello, @lunaticsm 

For CLI on Linux you can create hey config ".config/hey/conf.toml" and write in it:
```
model = "GPT4o"
```
https://github.com/mrgick/duck_chat/blob/f448d826cb746647ec25911a0745f5d0ee707b38/duck_chat/cli.py#L141

If you are using as lib
```python
import asyncio
from duck_chat import DuckChat, ModelType

async def main():
    async with DuckChat(model=ModelType.GPT4o) as chat:
        print(await chat.ask_question("2+2?"))
        await asyncio.sleep(1)
        print(await chat.ask_question("6+6?"))

asyncio.run(main())
```

**Comment by lunaticsm** [comment]
Thanks! i'm using as lib, btw is it possible to generate an image ?

**Comment by mrgick** [comment]
Unfortunately, duck.ai with gpt-4o-mini doesn't provide image generation yet, so you won't be able to generate an image =(

**Comment by lunaticsm** [comment]
yep i just tried, so last question, is it possible to stream output ?

**Comment by mrgick** [comment]
Nope
There's aiohttp being used there, and when I tried to give the stream, it gave me already complete output, so I just converted it to a message. 
I maybe missed something then, but for now it's being given as an already finished message.

**Comment by mrgick** [comment]
Also, you can try by yourself to change it to stream
https://github.com/mrgick/duck_chat/blob/f448d826cb746647ec25911a0745f5d0ee707b38/duck_chat/api.py#L88
